src/utils/__tests__/__snapshots__/publicAPI.test.js.snap

```diff
@@ -5285,9 +5285,9 @@ Map {
         "rollingPeriodLabel": "Rolling period",
         "searchPlaceholder": "Search",
         "selectAllAria": "Select all items",
-        "selectDateRangeLabel": "Select time range",
```

Comment:
To avoid a breaking change we need to leave this in and deprecate it with a warning and map it to the new `selectTimeRangeLabel ` prop.